shennah_al-jawad/week_01/Weds-Homework.html

```diff
@@ -0,0 +1,50 @@
+<html>
+<body>
+<script>
+
+//The Word Guesser
+var word = ['D', 'O', 'G'];
+var guesses = ['_', '_', '_',];
+var counter = word.length;
+console.log(guesses);
+console.log("Let the games begin!")
+
+function guessLetter(l) {
+
+	var correctGuess = false;
+	var incorrectGuess = false;
+
+	for (var x = 0; x < word.length; x++) {
+		if (word[x] === l) {
+			guesses[x] = l;
+			correctGuess = true;
+			counter = counter + 1;
+			console.log(guesses)
+		}
+		else if (word[x] === '_') {
+			incorrectGuess = true;
+		}
+	}
+
+	if (correctGuess === true) {
+		console.log("You found a letter!");
+		if (counter === 3) {
```

Comment:
here you might want instead of hardcoding number 3, put smth like word.length, so it become more flexible :)